scripts/update-ips.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+: "${CK8S_CONFIG_PATH:?Missing CK8S_CONFIG_PATH}"
+CK8S_CONFIG_PATH=$(readlink -f "${CK8S_CONFIG_PATH}")
+here="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+
+## Add object storage ips
+S3_ENDPOINT="$(yq4 '.objectStorage.s3.regionEndpoint' "${CK8S_CONFIG_PATH}/common-config.yaml" | sed 's/https\?:\/\///')"
+if [[ "${S3_ENDPOINT}" == "" ]]; then
+    echo "No S3 endpoint found, check your common-config.yaml" >&2
+    exit 1
+fi
+
+mapfile -t IPS < <(dig +short "${S3_ENDPOINT}")
+if [ ${#IPS[@]} -eq 0 ]; then
+    echo "No ips for s3 was found (tried looking at DNS for ${S3_ENDPOINT})" >&2
+    exit 1
+fi
+
+yq4 -i '.networkPolicies.global.objectStorage.ips = []' "${CK8S_CONFIG_PATH}/common-config.yaml"
+for ip in "${IPS[@]}"; do
+    yq4 -i '.networkPolicies.global.objectStorage.ips |= . + ["'"${ip}"'/32"]' "${CK8S_CONFIG_PATH}/common-config.yaml"
+done
+
+## Add sc ingress ips
+OPS_DOMAIN="$(yq4 '.global.opsDomain' "${CK8S_CONFIG_PATH}/common-config.yaml")"
+if [[ "${OPS_DOMAIN}" == "" ]]; then
+    echo "No ops domain found, check your common-config.yaml" >&2
+    exit 1
+fi
+
+mapfile -t IPS < <(dig +short "grafana.${OPS_DOMAIN}")
+if [ ${#IPS[@]} -eq 0 ]; then
+    echo "No ips for SC ingress found (tried looking at DNS for grafana.${OPS_DOMAIN})" >&2
+    exit 1
+fi
+
+yq4 -i '.networkPolicies.global.scIngress.ips = []' "${CK8S_CONFIG_PATH}/wc-config.yaml"
+for ip in "${IPS[@]}"; do
+    yq4 -i '.networkPolicies.global.scIngress.ips |= . + ["'"${ip}"'/32"]' "${CK8S_CONFIG_PATH}/wc-config.yaml"
+done
+
+## Add wc ingress ips
+BASE_DOMAIN="$(yq4 '.global.baseDomain' "${CK8S_CONFIG_PATH}/common-config.yaml")"
+if [[ "${OPS_DOMAIN}" == "" ]]; then
+    echo "No base domain found, check your common-config.yaml" >&2
+    exit 1
+fi
+
+# Try to fetch the ips for *.${BASE_DOMAIN}, which should point to wc ingress
+mapfile -t IPS < <(dig +short "non-existing-subdomain.${BASE_DOMAIN}")
+if [ ${#IPS[@]} -eq 0 ]; then
+    echo "No ips for WC ingress found (tried looking at DNS for non-existing-subdomain.${BASE_DOMAIN})" >&2
+    exit 1
+fi
+
+yq4 -i '.networkPolicies.global.wcIngress.ips = []' "${CK8S_CONFIG_PATH}/sc-config.yaml"
+for ip in "${IPS[@]}"; do
+    yq4 -i '.networkPolicies.global.wcIngress.ips |= . + ["'"${ip}"'/32"]' "${CK8S_CONFIG_PATH}/sc-config.yaml"
+done
+
+## Add sc apiserver ips
+mapfile -t IPS < <("${here}/../bin/ck8s" ops kubectl sc get node -l node-role.kubernetes.io/control-plane= -ojsonpath='{.items[*].status.addresses[?(@.type=="InternalIP")].address}')
+if [ ${#IPS[@]} -eq 0 ]; then
+    echo "No ips for sc control-plane nodes found" >&2
+    exit 1
+fi
+
+yq4 -i '.networkPolicies.global.apiserver.ips = []' "${CK8S_CONFIG_PATH}/sc-config.yaml"
+for ip in "${IPS[@]}"; do
+    yq4 -i '.networkPolicies.global.apiserver.ips |= . + ["'"${ip}"'/32"]' "${CK8S_CONFIG_PATH}/sc-config.yaml"
```

Comment:
```suggestion
yq4 -i '.networkPolicies.global.scApiserver.ips = []' "${CK8S_CONFIG_PATH}/sc-config.yaml"
for ip in "${IPS[@]}"; do
    yq4 -i '.networkPolicies.global.scApiserver.ips |= . + ["'"${ip}"'/32"]' "${CK8S_CONFIG_PATH}/sc-config.yaml"
```

Reply:
Right, this file should've been deleted.